packages/data-point-express/lib/inspector-middleware.js

```diff
@@ -12,7 +12,7 @@ const InspectorUi = require('./inspector-ui')
 function dataPointInspectRoute (dataPoint, req, res, next) {
   const { entityId, params = {}, query = {}, value } = req.body
 
-  const pathname = url.parse(req.url).pathname
```

Comment:
Updating this was not straight forward, because of this I added the eslint-disable-line and created an issue to address individually https://github.com/ViacomInc/data-point/issues/352